.travis.yml

```diff
@@ -15,9 +15,10 @@ script:
   - docker-compose -f docker-compose.yml -f docker-compose-test.yml run wrangler
   - docker-compose -f docker-compose.yml -f docker-compose-test.yml run job-runner
   - docker-compose -f docker-compose.yml -f docker-compose-test.yml run patient-care
-  # Then, spin them up normally in daemon mode and confirm they run
-  - docker-compose -f docker-compose.yml up -d
   - docker ps
+  ## Then, spin them up normally in daemon mode and confirm they run
+  #- docker-compose -f docker-compose.yml up -d
```

Comment:
Just as I expected. After removing all patient-care app code changes from the project, the patient-care testing step no longer takes 1000+ seconds. This means we can put back this step in `travis.yml`.